src/main/cpp/subsystems/SwerveDrive.cpp

```diff
@@ -327,36 +337,36 @@ bool SwerveDrive::DriveAuto(double max_speed) {
 }
 
 bool SwerveDrive::AutoBalance() {
```

Comment:
It would be good to pass a direction argument into this function because then it would be easy to flip the signs and do this from either side of the charge station